graylog2-rest-client/src/test/java/org/graylog2/restclient/lib/ApiClientTest.java

```diff
@@ -59,6 +59,16 @@ public void testBuildTarget() throws Exception {
                 "query=+%22.%2B%22",
                 queryParamWithDoubleQuotes.getQuery());
 
+        final URL queryParamWithColon = api.get(EmptyResponse.class).path("/some/resource").queryParam("query", "mac:00\\:12\\:f0\\:53\\:42\\:2b").node(node).unauthenticated().prepareUrl(node);
+        Assert.assertEquals("query param with \\ should be escaped",
```

Comment:
I wanted to ensure that this odd case we found when trying to migrate to okhttp was still working as expected: https://github.com/Graylog2/graylog2-web-interface/issues/1356